frontend/src/app/map-viewer/map-viewer.component.ts

```diff
@@ -0,0 +1,42 @@
+import {AfterViewInit, Component, Input, OnDestroy, ViewChild} from '@angular/core';
+import {GoogleMaps} from "./google-maps";
+import {MapMarker} from "./map-marker";
+import {BehaviorSubject} from "rxjs/index";
+
+@Component({
+  selector: 'app-map-viewer',
+  templateUrl: './map-viewer.component.html',
+  styleUrls: ['./map-viewer.component.scss']
+})
+export class MapViewerComponent implements AfterViewInit, OnDestroy {
```

Comment:
No unit tests for this component?

Reply:
It's no easy to test this component.
Its integrate with Google maps and display makers .

Reply:
ok :(